partiql-ast/src/ast.rs

```diff
@@ -64,33 +66,38 @@ pub enum DdlOp {
     DropIndex(DropIndex),
 }
 
-#[derive(Clone, Debug, PartialEq, Eq)]
+#[derive(Visit, Clone, Debug, PartialEq, Eq)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub struct CreateTable {
+    #[visit(skip)]
```

Comment:
Out of curiosity, why are some `ast` nodes skipped?

Reply:
Importantly, it's not really 'ast nodes' that are skipped, but rather data items that inhere to the current node.

In some sense, we recursively visit 'children'. Things that aren't 'children' get 'skip'ed by the visitor macros, as it wouldn't really make sense for your visitor to have a `visit_symbol_primitive` method that is divorced from what node that symbol is appearing in.